webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala

```diff
@@ -2366,7 +2366,7 @@ object HierarchicalListValueContentV2 {
                .fail(BadRequestException(s"List node IRI <$listValueAsListNode> is not a Knora data IRI"))
                .when(!listValueAsListNode.isKnoraDataIri)
         comment <- JsonLDUtil.getComment(jsonLDObject)
-      } yield HierarchicalListValueContentV2(ApiV2Complex, listValueAsListNode.toString, comment)
```

Comment:
this is the actual fix, the comment `Option[String]` was passed in as the label.